test/simulator/timestamp/src/connection_simulator.cpp

```diff
@@ -168,6 +166,11 @@ connection_simulator::decode_timestamp_config_map(std::map<std::string, std::str
         has_durable = true;
         config_map.erase(pos);
     }
+    pos = config_map.find("force");
+    if (pos != config_map.end()) {
+        force = true;
+        config_map.erase(pos);
+    }
```

Comment:
Should we explicitly set `force` to `false` in an `else` statement?

Reply:
It is set to false [here](https://github.com/wiredtiger/wiredtiger/pull/8186/files#diff-402b40f7efe9213e945ecef68f5de126368d636319b6456a35145ff1c4725b00R191)